src/extension/VSProject.cs

```diff
@@ -125,9 +125,10 @@ public TestPackage GetTestPackage(string configName)
                 {
                     var config = _configs[name];
                     package.AddSubPackage(new TestPackage(config.AssemblyPath));
-                    appbase = config.OutputDirectory;
+                    appbase = config.OutputDirectory;
+
                     break;
-                }
+                }

```

Comment:
Hard to review a fix with a lot of unrelated formatting changes. I don't want you to go back and change it, but please be aware in future of the problems that reviewers face when you do this. 😄

Reply:
Sorry, I didn't noticed mess made by VS :-(

Reply:
I usually view all diffs before each commit I make. It is a pain though when you run into a file that doesn't follow our stated coding standard. That's on my todo list now.